source/overview/license-and-subscription.rst

```diff
@@ -71,13 +71,33 @@ You may incur retroactive charges for any unique users added that exceed the lic
 Renewing an Enterprise license
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The ability to renew your license in the Customer Portal will be available in a future release.
+You can renew your Mattermost license with a credit card if you have a standard Mattermost contract. When you renew your license, you can also increase the number of active users.
 
-To renew your license in the meantime, contact your Customer Success Manager to start the renewal process. Alternatively, `contact us <https://mattermost.com/contact-us/>`_ for assistance.
+If you are a reseller, have a non-standard contract, or want to adjust the number of active users on your license please contact sales@mattermost.com.
+
+System Admins will be alerted 60 days prior to the license expiry via a banner in Mattermost. Select **Renew license now** to start the renewal process in the Customer Portal. You can also dismiss the banner and renew your license at a later date via **System Console > Edition and License**.
+
+The renewal page in the Customer Portal lists your license information and contact details which is pre-populated based on your existing license subscription. 
+
+**To process your license renewal:**
+
+1. Enter your contact and billing information.
+2. Confirm the Mattermost Enterprise Edition version.
+3. Confirm the listed number of active users is correct.
+4. Select **Complete purchase**. 
+
+An email with the new license key and information on how to upload the license in the System Console will be sent to the primary email address.
+
+As a final step you can optionally, you can create a Customer Portal account to manage your license subscriptions. This account is linked to your license and not your usrname or user ID, and can be accessed by other Mattermost System Admins.
 
 Frequently Asked Questions
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+How do I renew my license if I don't have internet access?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you don't have access to the internet, we'll assist you via email.
```

Comment:
Is there a specific email address we can provide, or sales@mattermost.com? @jasonblais

Reply:
`support@mattermost.com`